tests/test_typescript_features.py

```
@@ -0,0 +1,524 @@
+    """Test the 'impact' (reverse call graph) command for TypeScript."""
+
+    def test_impact_finds_callers(self, temp_ts_project):
+        """impact should find all callers of a function."""
+        tmpdir = temp_ts_project({
+            "main.ts": MULTI_FILE_CALLER,
+            "helper.ts": MULTI_FILE_HELPER,
+        })
+        result = run_tldr(["impact", "helper", "--project", tmpdir])
+
+        # main.ts calls helper, so it should appear in impact
+        output = json.dumps(result)
```

Ruff (0.14.11)

Local variable `output` is assigned to but never used

Remove assignment to unused variable `output`

(F841)